### PR TITLE
CPP Solution to Icy Perimeter

### PR DESCRIPTION
**Pull request by aldew5**
Revised

### CODE REVIEW

**Comment by maggie-j-liu** [comment]
> Please follow the steps [here](https://usaco.guide/general/contributing?lang=cpp#adding-a-solution), in particular, make sure to modify the problem constructor and only use the macros mentioned there.
> Also, use tabs instead of spaces.

Your solution for revegetate is also included; could you delete it?

**Comment by aldew5** [comment]
@maggie-j-liu the other file has been deleted.  In terms of updating the constructor, I looked at it and wasn't sure what to include.  The step shown in the contributing section of the guide seemed to already have been completed.  Also, I did you use tabs.  I think when I pasted my code into the template it formatted as spaces.  I can probably figure out how to correct this if that's necessary.

**Review by andrewwangva — CHANGES_REQUESTED** [review]

**Comment by dongliu0426** [comment]
marking this as a draft for now; open this after the requested changes are fixed.

File: solutions/USACO feb19-perimeter-silver.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+id: usaco-895
+source: USACO Silver 2019 January
+title: Icy Perimeter
+author: Alec Dewulf 
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_perimeter_silver_jan19.html)
+
+This problem can be solved using [Flood Fill](/silver/flood fill) (a Silver topic).
```
[the remark this answers is not in the document]
Comment (jessechoe10) [reply solutions/USACO feb19-perimeter-silver.mdx:10]: We don't need this unless you're addressing a topic from a higher division

**Comment by jessechoe10** [comment]
I'm gonna close this PR for now. We can implement your PR once you address the changes.

**Pull request closed without merging**